CMakeLists.txt

```diff
@@ -1,3 +1,26 @@
 cmake_minimum_required(VERSION 2.8)
 add_subdirectory(generator)
-add_subdirectory(indexgenerator)
\ No newline at end of file
+add_subdirectory(indexgenerator)
+
+INSTALL(
+  DIRECTORY data
+  DESTINATION ${CMAKE_INSTALL_PREFIX}/share/woboq_codebrowser
```

Comment:
Does it make sens to install the `data`  there?
It is only meant to be served by the http server.